### DISCUSSION TITLE
Textual in REPLIT?

### DISCUSSION THREAD

**Discussion by JeremyBYU** [Textualize/textual#2132] [Q&A]
Hi everyone,
I was wondering if anyone has had success using textual inside of [REPLIT](https://replit.com/~). Everything installs but the output seems malformed. For example, this is the output of `python -m textual`:

![image](https://user-images.githubusercontent.com/6628085/227519306-a4bf2c2e-dd33-43bb-9bbd-de457676f9e1.png)

My guess is that the built in "terminal" is not supported. 

Thanks!

**Comment by davep** [comment]
My understanding is that it's not so much that Textual doesn't "support" the terminal that's used there; it's that the terminal that's used there isn't quite as capable as a "normal" terminal on your desktop.

**Comment by willmcgugan** [comment]
I have had Textual working in REPLIT. It was a little slow and a few glitches, but otherwise looked fine (better than those screenshots). Will investigate at some point.

**Reply by JeremyBYU** [reply]
Thanks for looking into it!

**Comment by iamsirsammy** [comment]
Textual worked and looked fantastic on replit late last year, but looks like the screenshot now. There is also a dependency conflict with replit and rich. I would love to see this incompatibility fixed at some point as well
